addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -153,13 +154,15 @@ def _l10n_it_post_invoices_step_1(self, invoices):
                 'error': _("You must accept the terms and conditions in the settings to use FatturaPA."),
                 'blocking_level': 'error'} for invoice in invoices}
 
-        if self.env['ir.config_parameter'].get_param('account_edi_proxy_client.demo', False):
-            responses = {filename: {'id_transaction': 'demo'} for invoice in invoices}
-        else:
+        demo_state = self.env['ir.config_parameter'].get_param('account_edi_proxy_client.demo', False)
```

Comment:
we could also have a tiny method _l10n_it_get_demo_state api.model which could return test/demo/prod so we don't need to repeat if demo_state in...  (I first thought it as not too bad, but we have it like 7 times)